api/src/controllers/platform/stats.ts

```diff
@@ -99,14 +104,16 @@ export async function platformStats(
     ...space,
     users: users.map((v) => ({
       activationLink: find.activationLink(v.activationKey),
-      createdAt: v.createdAt,
+      createdAt: v.createdAt as unknown as string,
       email: v.email,
       fullname: v.fullname,
-      lockedAt: v.lockedAt,
+      lockedAt: v.lockedAt as unknown as string,
       resetKeyLink: find.resetKeyLink(v.resetKey),
-      resetKeyCreatedAt: find.resetKeyCreatedAt(v.resetKeyExpiresAt),
-      resetKeyExpiresAt: v.resetKeyExpiresAt,
-      role: v.platformRole,
+      resetKeyCreatedAt: find.resetKeyCreatedAt(
+        v.resetKeyExpiresAt
+      ) as unknown as string,
+      resetKeyExpiresAt: v.resetKeyExpiresAt as unknown as string,
+      role: v.platformRole as unknown as ETeamRole,
```

Comment:
Still a bit surprised about these `as unknown as type`. I wonder if there is a way to avoid them.

Reply:
Marking this issue as resolved in favor of your earlier comment. We can discuss it there.